crates/db_views/src/site/vote_view.rs

```diff
@@ -176,6 +176,8 @@ mod tests {
         score: 1,
       },
     ];
+    expected_post_vote_views[1].creator.post_count = 1;
+    expected_post_vote_views[1].creator.comment_count = 1;
```

Comment:
I'm guessing this is due to the currently broken triggers.

Reply:
No its because these were previously not covered by tests at all as it only compared `Person`, not `PersonAggregates`. After moving the aggregates into `Person` I have to adjust the counts after voting to make the test pass.

Reply:
Gotcha.